packages/angular/cli/models/schematic-engine-host.ts

```diff
@@ -115,10 +120,8 @@ function wrap(
   moduleCache: Map<string, unknown>,
   exportName?: string,
 ): () => unknown {
-  const { createRequire, createRequireFromPath } = require('module');
-  // Node.js 10.x does not support `createRequire` so fallback to `createRequireFromPath`
-  // `createRequireFromPath` is deprecated in 12+ and can be removed once 10.x support is removed
-  const scopedRequire = createRequire?.(schematicFile) || createRequireFromPath(schematicFile);
+  const { createRequire } = require('module');
```

Comment:
nit: This can be an import now.

Reply:
This module can only be referenced with ECMAScript imports/exports by turning on the 'esModuleInterop' flag and referencing its default export.